Description: Drop rack-test dependency
 The dependency on rack-test is only relevant if using ruby 2.1.
 As we're only targeting suites which ship at least ruby 2.3, drop it
 accordingly. 
Author: Georg Faerber <georg@riseup.net>
Last-Update: 2018-05-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/schleuder.gemspec
+++ b/schleuder.gemspec
@@ -33,8 +33,6 @@
   s.add_runtime_dependency 'mail', '~> 2.6.0'
   s.add_runtime_dependency 'mail-gpg', '~> 0.3', '>= 0.3.3'
   s.add_runtime_dependency 'activerecord', '~> 4.2'
-  # TODO: Drop this once we cease to support ruby 2.1, see #310
-  s.add_runtime_dependency 'rack-test', '~> 0.7.0'
   s.add_runtime_dependency 'rake', '>= 10.5.0'
   s.add_runtime_dependency 'sqlite3', '~> 1'
   s.add_runtime_dependency 'sinatra', '~> 1'
